File: hw/collahuazo/u3/hw24CompositePaterns/CompositePatternJS/composite/model/Manager.js
const { Supervisor } = require("./Supervisor.js");
class Manager extends Supervisor {
  constructor(aName) {
    super();
    this.title = "Manager";
    this.name = aName;
  }

  stateName() {
    super.stateName();
  }
}

module.exports = { Manager };

File: hw/collahuazo/u3/hw24CompositePaterns/CompositeEmployee/src/main/java/ec/edu/espe/composite/model/Manager.java
package ec.edu.espe.composite.model;

/**
 *
 * @author Brandon Collahuazo,CodeBros,@ESPE
 */
public class Manager extends Supervisor {

    public Manager(String aName) {
        this();
        name = aName;
    }

    public Manager() {
        super();
        title = "Manager";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}
